.changeset/five-plums-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@solid-primitives/websocket": patch
```

Comment:
this doesn't look like a patch lol
Why not make it a `major` since it's a breaking rewrite?
```suggestion
"@solid-primitives/websocket": major
```